.changeset/rotten-mugs-smash.md

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
`rotten-mugs-smash.md`?!

Reply:
it's a randomly generated doc from changsets  https://github.com/changesets/changesets/blob/main/docs/common-questions.md#each-changeset-is-its-own-file